java/ql/src/Security/CWE/CWE-023/PartialPathTraversal.qhelp

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE qhelp PUBLIC
+  "-//Semmle//qhelp//EN"
+  "qhelp.dtd">
+<qhelp>
+<overview>
+<p>User supplied file paths can often pose security risks if a program does not correctly handle them. In particular, if a user 
+is meant to access files under a certain directory but does not enters a path under that directory, they can gain access to 
+(and potentially modify/delete) unexpected, possibly sensitive resources. </p>
```

Comment:
unexpectedly

Reply:
Should it then be "...they can unexpectedly gain access to (and potentially modify/delete) unexpected, possibly sensitive resources"?
(I've currently phrased it that way, but let me know if you'd prefer something different!)

Reply:
You can do both here, choice is yours :-)